scripts/prepare_release.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
Can you pls update https://github.com/PyGithub/PyGithub/blob/main/DEPLOY.md as well?

Reply:
Done